resolwe_bio/processes/differential_expression/deseq.yml

```diff
@@ -23,6 +23,12 @@
       type: list:data:expression
       description: >
         Control samples (replicates)
+    - name: filter
+      label: Filtering of raw counts
+      type: basic:integer
+      default: 10
+      description: >
+        The threshold value for raw counts filtering.
```

Comment:
Write what it does. For example, "Filter genes in the expression matrix input. Remove genes where the number of counts in all samples is below the threshold.".